coalib/coala_modes.py

```diff
@@ -0,0 +1,92 @@
+from coalib.coala_main import run_coala
```

Comment:
this import is huge and slow, deferring it into the functions will yield a bigger perf benefit that all the other imports I think